Makefile

```diff
@@ -100,7 +100,11 @@ else
 	JSON=$(JSON) VERSION=$(VERSION) scripts/makejson
 endif
 
-clean:
+# Clean files produced by "wad-diff".
+wad-diff-clean:
+	rm -f wad-images/*.png
```

Comment:
I was just trying to make my changes as isolated as possible, but I'd be happy to add to the main target.